### PR TITLE
Dashboard's Farm Item, Login and Signup components

### LINKED ISSUE #15: [FEATURE] Create Farm Item Dashboard Component for web ui
Is your feature request related to a problem? Please describe.
Create a dashboard component for the react web app, where users are routed after successful login

Describe the solution you'd like
Should use material design consistent with the dev spec
Should follow the design of the created mockup

Additional context
Branch name: ft-dashboard-web-ui

### PR DESCRIPTION
**Pull request by MariaAngella**
## Description
Dashboard's Farm Item component, Login and Signup components

Fixes #15 #13

## How Has This Been Tested?
Work In Progress

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added necessary inline code documentation
- [ ] I have added tests that prove my fix is effective and that this feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/web_ui/src/components/AppRoutes/constants.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  doLogin: "DO_LOGIN",
+  doLogout: "DO_LOGOUT",
+  doSearch: "DO_SEARCH",
+};
+
+export const localRoutes = {
+  dashboard: "/dashboard",
+  forum: "/forum",
+  info: "/info",
+  wholesalers: "/customers/wholesalers",
```
Comment (kmwamasali) [inline src/web_ui/src/components/AppRoutes/constants.js]: change this to 
`wholesalers: "/stalls/wholesalers"`

add 
`vendors: "/stalls/vendors"`

File: src/web_ui/src/components/Layout/LayoutDashboard/coreActions.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import {coreConstants} from "./coreReducer";
```
Comment (kmwamasali) [inline src/web_ui/src/components/Layout/LayoutDashboard/coreActions.js]: remove this file

File: src/web_ui/src/components/Layout/LayoutDashboard/coreReducer.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
+import {AUTH_TOKEN_KEY, AUTH_USER_KEY} from "../../AppRoutes/constants";
```
Comment (kmwamasali) [inline src/web_ui/src/components/Layout/LayoutDashboard/coreReducer.js]: remove this file

File: src/web_ui/src/components/Pages/DashboardComponents/FarmItem.js [hunk 4]
```diff
@@ -0,0 +1,154 @@
+import React from 'react';
```
Comment (kmwamasali) [inline src/web_ui/src/components/Pages/DashboardComponents/FarmItem.js]: transfer this to `components/Dashboard`

File: src/web_ui/src/components/Layout/LayoutDashboard/styles.js [hunk 5]
```diff
@@ -0,0 +1,99 @@
+import grey from "@material-ui/core/colors/grey";
```
Comment (kmwamasali) [inline src/web_ui/src/components/Layout/LayoutDashboard/styles.js]: transfer this to `components/Dashboard`

File: src/web_ui/src/components/Layout/LayoutDashboard/stringHelpers.js [hunk 6]
```diff
@@ -0,0 +1,41 @@
+export const getInitials = (fullName = '') => {
```
Comment (kmwamasali) [inline src/web_ui/src/components/Layout/LayoutDashboard/stringHelpers.js]: transfer this to `src/utils`

**Review by kmwamasali — CHANGES_REQUESTED** [review]
make the requested changes.
thanks

File: src/web_ui/src/components/AppRoutes/constants.js (hunk 1 above)
Comment (MariaAngella) [reply src/web_ui/src/components/AppRoutes/constants.js]: Done

File: src/web_ui/src/components/Layout/LayoutDashboard/coreActions.js (hunk 2 above)
Comment (MariaAngella) [reply src/web_ui/src/components/Layout/LayoutDashboard/coreActions.js]: Done

File: src/web_ui/src/components/Layout/LayoutDashboard/coreReducer.js (hunk 3 above)
Comment (MariaAngella) [reply src/web_ui/src/components/Layout/LayoutDashboard/coreReducer.js]: Done

File: src/web_ui/src/components/Pages/DashboardComponents/FarmItem.js (hunk 4 above)
Comment (MariaAngella) [reply src/web_ui/src/components/Pages/DashboardComponents/FarmItem.js]: Done

File: src/web_ui/src/components/Layout/LayoutDashboard/styles.js (hunk 5 above)
Comment (MariaAngella) [reply src/web_ui/src/components/Layout/LayoutDashboard/styles.js]: Done

File: src/web_ui/src/components/Layout/LayoutDashboard/stringHelpers.js (hunk 6 above)
Comment (MariaAngella) [reply src/web_ui/src/components/Layout/LayoutDashboard/stringHelpers.js]: Done

**Comment by MariaAngella** [comment]
Done as per the reviews suggested

**Pull request merged**